embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/network/http/EmbraceUrlStreamHandler.java

```diff
@@ -119,4 +119,17 @@ protected void injectTraceparent(@NonNull URLConnection connection) {
             connection.addRequestProperty(TRACEPARENT_HEADER_NAME, embrace.generateW3cTraceparent());
         }
     }
+
+    /**
+     * If sdk is not started we won't wrap url connection. Instead, we return the original connection.
+     * @param connection
+     * @return
+     */
+    private URLConnection newUrlConnection(URLConnection connection) {
```

Comment:
Maybe call this `wrapURLConnection` since it's not always creating a new one?

Reply:
done